projects/core/components/data-list/data-list.component.ts

```diff
@@ -133,7 +133,9 @@ export class TuiDataListComponent<T>
 
     private get elements(): readonly HTMLElement[] {
         return Array.from(
-            this.el.querySelectorAll('a[tuiOption],button[tuiOption],label[tuiOption]'),
+            this.el.querySelectorAll(
```

Comment:
Maybe just query `[tuiOption]` attr?

Reply:
https://github.com/taiga-family/taiga-ui/pull/10431/files

@vladimirpotekhin Could you help me? Why is it `a[tuiOption],button[tuiOption],label[tuiOption]` instead of `[tuiOption]`?